ads/_config.js

```diff
@@ -735,6 +735,10 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
+  'opinary': {
+    renderStartImplemented: true,
```

Comment:
See my comment above. Is this supported on your server? If not, set this to false, and you can add this later. If so, please make a fix and I can test again. 😄

Reply:
Firstly, thank you for the speedy review!

You are absolutely right about `renderStartImplemented`, this is not supported on our server so I have set it to false for now :)

Reply:
@tyroneopinary

You are welcome! 😄 Did you plan on implementing `renderStart` on your server soon? If not, we really encourage it. It helps us serve ads as fast as possible to our users, and gives us some big performance gains.

If you don't plan on implementing soon, feel free to just remove the line.

Otherwise, if you do plan on implementing soon, we can merge this, and you can update the 
 line to true in another PR.

Thank you! 👍

Reply:
Understood.

I've removed the line in question now, and will add it back when it is supported.

Cheers :)